reporter/src/main/kotlin/model/EvaluatedModel.kt

```diff
@@ -108,8 +109,11 @@ data class EvaluatedModel(
     val ruleViolationResolutions: List<RuleViolationResolution>,
     val ruleViolations: List<EvaluatedRuleViolation>,
     val statistics: Statistics,
-    // TODO: Ideally this would be an instance of RepositoryConfiguration, but for now it has to be a string to not be
-    //       converted to JSON when using it as input for the web app reporter.
+    val repository: Repository,
+    /**
```

Comment:
Nit: Now that we have KDoc, we usually group it to the property by adding a blank line before this one, and a blank line after the property.

Reply:
Done.